src/component/feature/FeatureToggleList/FeatureToggleListTable/FeatureToggleListTable.tsx

```diff
@@ -22,12 +22,15 @@ import { ConditionallyRender } from 'component/common/ConditionallyRender/Condit
 import { PageContent } from 'component/common/PageContent/PageContent';
 import { PageHeader } from 'component/common/PageHeader/PageHeader';
 import { sortTypes } from 'utils/sortTypes';
+import { usePersistentQuery } from 'hooks/usePersistentQuery';
 
 interface IExperimentProps {
     data: Record<string, any>[];
     isLoading?: boolean;
 }
 
+type IPageQuery = Partial<Record<'q' | 'sort' | 'order', string>>;
```

Comment:
I used `q`, because I frequently see it as a parameter. 
- [google.com**/search?q=getunleash](https://www.google.com/search?q=getunleash)
- [stackoverflow.com/search?q=unleash](https://stackoverflow.com/search?q=unleash)
But maybe `?search` is better?
- [en.wikipedia.org/w/index.php?search=feature%20flags](https://en.wikipedia.org/w/index.php?search=feature%20flags)